src/types/onyx/PolicyTag.ts

```diff
@@ -3,11 +3,17 @@ type PolicyTag = {
     name: string;
 
     /** Flag that determines if a tag is active and able to be selected */
-    enabled: boolean;
+    enabled?: boolean;
```

Comment:
Why did this become optional? AFAIK tags always have this property, where did we see that this is not the case?